frontend/js/components/Layout.js

```diff
@@ -2,12 +2,84 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 class Layout extends Component {
+
+    titlecase(str) {
+        return str.replace(/\w\S*/g, (txt) => {
+            return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+        });
+    }
+
+    candidateMatchString(matches) {
+        if (matches.length === 1) {
+            return matches[0].name;
+        }
+        if (matches.length === 2) {
+            return `${matches[0].name} and ${matches[1].name}`;
+        }
+        if (matches.length === 3) {
+            return `${matches[0].name}, ${matches[1].name}, and ${matches[2].name}`;
+        }
+    }
+
+    getSocialMediaMetaTags() {
+
+        const bestMatches = this.props.state.result.bestMatches;
+        const baseUrl = 'http://okcandiate.code4hr.org';
```

Comment:
@wbprice is there a way we can set `baseUrl` to the actual base url rather than hard coding it? Would be make the code more transportable.